rskj-core/src/main/java/co/rsk/peg/federation/FederationSupport.java

```diff
@@ -53,5 +53,7 @@ public interface FederationSupport {
 
     void updateFederationCreationBlockHeights();
 
+    BtcTransaction createAndSetSvpFundTransactionWithoutSignatures(Coin spendableValueFromProposedFederation, Coin feePerKb, Wallet activeFederationWallet) throws InsufficientMoneyException;
```

Comment:
Maybe the `spendableValueFromProposedFederation` should come from BridgeConstants?

Reply:
good one. Thanks.